docs/docs/tour/tour.md

```diff
@@ -0,0 +1,152 @@
+# Tour
+
+Once you've started the DBSP demo, as described in [Get
+Started](../intro), navigate to [localhost:8085](http://localhost:8085)
+in your web browser to view the DBSP user interface.  A status
+dashboard fills most of the home screen, as shown below, with a
+sidebar on the left offering access to tabs for DBSP features:
+
+![DBSP home screen](home.png)
+
+Let's take a quick tour around the options on the sidebar.
+
+## Analytics
+
+![Analytics](analytics.png)
+
+The top group of tabs allows a DBSP user to define the structure of
+data and specify analyses.  Both are these are written as SQL: data
+definitions as `CREATE TABLE` commands, and analyses as `CREATE VIEW`
+commands.  DBSP implements a rich SQL feature set that permits
+sophisticated SQL written for databases such as Postgres and MySQL to
```

Comment:
I would leave out specific database names, in fact there's a lot of Postgres stuff we don't support and may never be able to.